pkgs/development/libraries/tbb/default.nix

```diff
@@ -36,6 +36,13 @@ stdenv.mkDerivation rec {
       url = "https://github.com/oneapi-src/oneTBB/pull/833/commits/c18342ba667d1f33f5e9a773aa86b091a9694b97.patch";
       sha256 = "ZUExE3nsW80Z5GPWZnDNuDiHHaD1EF7qNl/G5M+Wcxg=";
     })
+
+    # Fixes build for aarch64-darwin
+    (fetchurl {
+      name = "aarch64-darwin.patch";
+      url = "https://github.com/oneapi-src/oneTBB/commit/86f6dcdc17a8f5ef2382faaef860cfa5243984fe.patch";
```

Comment:
Is this from an unmerged PR? Doesn't appear to be part of any branch in upstream repo.

Reply:
Actually I take this patch from homebrew: https://github.com/Homebrew/homebrew-core/blob/master/Formula/tbb@2020.rb#L27-L32, but it seems that this patch not from `oneapi-src/oneTBB` but somewhere else.

Reply:
Are we able to find a more stable source for this? Someone must surely have opened a PR for it when they created it.

Reply:
I've found it in forked repo: https://github.com/betajippity/oneTBB/commit/86f6dcdc17a8f5ef2382faaef860cfa5243984fe and in rejected PR to https://github.com/oneapi-src/oneTBB: https://github.com/oneapi-src/oneTBB/pull/258. PR rejected since in new version of `tbb` it uses CMake instead of Make and CMake build doesn't have this issue. I can change patch URL either to forked  https://github.com/betajippity/oneTBB repo or to PR, I'm okay with both variants.

Reply:
I would suggest setting it to `https://github.com/oneapi-src/oneTBB/pull/258/commits/86f6dcdc17a8f5ef2382faaef860cfa5243984fe.patch`, this way it's bound to the exact commit, but still conveys what its original source was.